frontend/src/queries/Query/Query.tsx

```diff
@@ -63,18 +54,14 @@ export function Query(props: QueryProps): JSX.Element | null {
     }
 
     let component
-    if (isLegacyQuery(query)) {
-        component = <LegacyInsightQuery query={query} context={queryContext} />
-    } else if (isDataTableNode(query)) {
+    if (isDataTableNode(query)) {
         component = (
             <DataTable query={query} setQuery={setQuery} context={queryContext} cachedResults={props.cachedResults} />
         )
     } else if (isDataNode(query)) {
         component = <DataNode query={query} cachedResults={props.cachedResults} />
     } else if (isInsightVizNode(query)) {
         component = <InsightViz query={query} setQuery={setQuery} context={queryContext} />
-    } else if (isInsightQueryNode(query)) {
```

Comment:
This doesn't have `legacy` in the name, is it right to remove this?

Reply:
Good spot 👀 ! It should be alright to remove this as it was only used by the "ad hoc" insight and I've checked plugins for usage of these components. Your comment led me to check the /debug page though, where the queries still exist and now I'm wondering wether we want to support rendering all source nodes as well. Will re-add this or remove the examples in a separate PR though.